data/json/monsters/bird.json

```
@@ -343,6 +352,7 @@
     "melee_cut": 3,
     "dodge": 6,
     "harvest": "bird_tiny",
+    "dissect": "dissect_bird_sample_small",
```

```suggestion
    "dissect": "dissect_chimera_sample_small",
```
I think counting cockatrice as a natural animal is probably a relic of the older game.